test/test_torch.py

```diff
@@ -7820,19 +7820,9 @@ def cholesky_test_helper(n, batch_dims, upper):
     @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
     @tf32_on_and_off(0.01)
     def test_cholesky(self, device, dtype):
```

Comment:
Would you move this test to test_linalg.py? It will need a new name, like "test_torch_cholesky" or "test_old_cholesky."

Reply:
Also, is there a change to this test that would demonstrate your lda fix?

Also also, if you spot other Cholesky tests let's move them, too.